Add a Travis check to ensure we remember to reset SQLITE_MAX_PAGE_COUNT

rafecolton opened the pull request:
cc @bondydaa @tylerkaraszewski 

### Details
If we update sqlite3.c, this value won't be updated in the mainline, so we need to ensure it gets updated or else we will run into problems.

### Fixed Issues
Part of https://github.com/Expensify/Expensify/issues/219373

### Tests
Added a check to the Travis tests, they should pass.
_________
**Internal Testing Reminder:** when changing bedrock, please compile auth against your new changes

bondydaa reviewed (approved):
looks like it worked https://app.travis-ci.com/github/Expensify/Bedrock/builds/253436994

Do we care that there's no output during successful runs? I think it's fine so approving.
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/4073354/180073817-b9e135f1-9c4d-42f7-9971-86af6e1e3502.png">

rafecolton commented:
I did notice that but I don't think it's a big deal. There will be output if it fails.

tylerkaraszewski commented:
I was thinking that instead of having travis fail if this is wrong, we'd make a build script for sqlite3.c that always does it correctly, and use that to generate our sqlite files.

rafecolton commented:
@tylerkaraszewski it does that in the build script on Jenkins for the CLI (separate PR), but building it for Bedrock here is somewhat manual. Until we script that part too, I added it to the SO instructions, which is hopefully sufficient. Added this check here because we all know nobody reads.

rafecolton commented:
I see your other comment about building it with that flag in the first place. I agree that would be better.

rafecolton commented:
@tylerkaraszewski I meant to put this on hold but didn't get to it fast enough. Do you think we should revert and include it in the build flags instead?

bondydaa commented:
I think that'd be worthwhile too but also think this is a nice easy win to
get in to ensure that we don't screw this up while getting the build script
figured out.

On Thu, Jul 21, 2022 at 10:26 AM Tyler Karaszewski ***@***.***>
wrote:

> I was thinking that instead of having travis fail if this is wrong, we'd
> make a build script for sqlite3.c that always does it correctly, and use
> that to generate our sqlite files.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Expensify/Bedrock/pull/1328#issuecomment-1191696197>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA7CPCTN64YNA426D5JLPLTVVF22XANCNFSM54EZUZCA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
